cookiecutter.json

```
@@ -14,6 +14,19 @@
   "license": ["MIT", "BSD-3", "GPL-3.0"],
   "funder": "JBFC: The Joe Bloggs Funding Council",
   "__prompts__": {
-    "deploy_docs_to_github_pages": "Enable automatically deploying HTML documentation to GitHub Pages website on pushes to main"
+    "author_given_names": "Given name(s) of package author",
+    "author_family_names": "Family name(s) of package author",
+    "author_email": "Email address for package author - will be publicly accessible",
```

```suggestion
    "author_email": "Email address for package author - will be part of package metadata.",
```
It could be possible to generate a package and put it in a private repo right, in which case the email wouldn't be publicly accessible?

Perhaps we should make the email optional if we're worried people won't want to add an email address because of publicly accessible reasons? But I can open a new issue for that.

---

Good point with regards to public versus private repository.